examples/opentracing/README.md

```diff
@@ -0,0 +1,82 @@
+# Overview
```

Comment:
What is the preferred format for readmes? I see that rst is used everywhere but in the main readme.

Reply:
Did not change this, but I agree. It is more consistent to use rst everywhere.

Reply:
I see, let's ignore this comment and go ahead.